src/client/pps.go

```diff
@@ -217,18 +221,84 @@ func (c APIClient) InspectJob(pipelineName string, jobID string, blockState bool
 	return jobInfo, grpcutil.ScrubGRPC(err)
 }
 
-// InspectJobOutputCommit returns info about a job that created a commit.
-// blockState will cause the call to block until the job reaches a terminal state (failure or success).
-func (c APIClient) InspectJobOutputCommit(repoName, branchName, commitID string, blockState bool) (*pps.JobInfo, error) {
-	jobInfo, err := c.PpsAPIClient.InspectJob(
-		c.Ctx(),
-		&pps.InspectJobRequest{
-			OutputCommit: NewCommit(repoName, branchName, commitID),
-			BlockState:   blockState,
-		})
+// BlockJob is a blocking version on InspectJob that will block
+// until the job has reached a terminal state.
+func (c APIClient) BlockJob(pipelineName string, jobID string, full ...bool) (*pps.JobInfo, error) {
```

Comment:
Added a separate call to wait for the job rather than a bool arg on `InspectJob` - similar to what we have for commits in the PFS client.